flowrs-build-designer/components/Rete.vue

```diff
@@ -13,12 +14,6 @@
 
 </template>
 
-<script setup lang="ts">
-import {useEventsStore} from "~/store/eventStore";
-
-const userStore = useEventsStore()
-const {isSaveButtonClicked} = storeToRefs(userStore)
-</script>
```

Comment:
warum löscht du hier stuff ?

Reply:
Das Ding macht keinen Sinn schau dir das mal im feature-Project1 branch an da wird doppelt <script> benutzt dieses Stück code ist aus meiner Sicht ohne nutzen.

Reply:
isSaveButtonClicked wird hier in einem setup tag gesetz 

das ist wichtig für den zweiten script tag der kein setup is

Reply:
Alles klar das war mir nicht bewusst ist jetzt wieder drin scheint aber auch ohne zu klappen.